README.md

```diff
@@ -96,11 +98,11 @@ The `start` method start (and restarts) mathjax-node. This allows reconfiguratio
 
 ### `typset(options, callback)`
 
-The `typeset` method is the main method of mathjax-node. It expects a configuration object and a callback.
+The `typeset` method is the main method of mathjax-node. It expects an configuration object `input` and a `callback`.
 
-Once started, `typeset` can be called repeatedly and will potentially store information across calls (cf. `state` below).
+Once called, `typeset` can be called repeatedly and will optionally store information across calls (see `state` below); this information can be reset using the `start()` method (see above).
```

Comment:
Note that a `start()` call won't reset the `state`.  You would need to set `state` to an empty `{}` to do that.  You can do that without calling `start()`.  (Calling `start()` is good for allowing a new configuration to be used, however.)